mlc_llm/utils.py

```diff
@@ -73,6 +73,8 @@ def argparse_postproc_common(args: argparse.Namespace) -> None:
         "open_llama": ("LM", "llama"),
         "rwkv-": ("rwkv", "rwkv"),
         "gorilla-": ("gorilla", "llama"),
+        "starcoder": ("code_gpt", "gpt_bigcode"),
```

Comment:
Can we make it `starcoder-` to align with other keys?

Reply:
Unfortunately left it with "starcoder", just because the [BigCode](https://huggingface.co/bigcode) StarCoder series start like that ("starcoder", "starcoderbase", "starcoderplus")

Reply:
got it, thank you for clarifying this!